src/container_inspector/cli.py

```diff
@@ -41,7 +41,8 @@ def container_inspector_squash(image_path, extract_directory):
 
 def _container_inspector_squash(image_path, extract_directory):
     images = get_images_from_dir_or_tarball(image_path)
-    assert len(images) == 1, 'Can only squash one image at a time'
+    if(len(images) == 1):   
```

Comment:
Is the code formatted? Is this if statement correct?
You need to have a hard look, and if need be, also add a test that passes before making this change.